docs/manual/spring-security-docs-manual.gradle

```diff
@@ -16,9 +16,10 @@ asciidoctor {
 asciidoctorj {
 	def docsTag = snapshotBuild ? 'current' : project.version
 	def ghTag = snapshotBuild ? 'main' : project.version
+	def samplesBranch = computeSamplesBranch(project.version as String)
```

Comment:
Can this be simplified to:

```groovy
def samplesBranch = project.version.replaceAll(/(\d+\.\d+\.).*/, '$1x');
```

Reply:
Yeah, and it cover more scenarios, thanks!